GOVERNANCE.md

```diff
@@ -244,7 +277,153 @@ The current NumFOCUS Subcommittee consists of:
 - Jon Sedar
 - Thomas Wiecki
 
-### Institutional Partners and Funding
+## Core Contributors
+Core Contributors are those individuals with commit or managing rights to any of
+the repositories or services of the PyMC Project. However, the exact permissions
+of all Core Contributors may not be the same and depend on their team memberships.
+Therefore, Core Contributors may act as Recurrent Contributors depending
+on the task at hand.
+We believe this to be necessary given the current size and scope of
+the PyMC project. Contributors may be proficient in some areas of the
+project while being inexperienced in some others.
+
+### Core Contributor membership
+To become a Core Contributor, one must already be a Recurring Contributor.
+Core Contributors can nominate any Recurring Contributor to become a
+Core Contributor. For the nomination to go forward, it has to be
+ratified by the Steering Council and the BDFL.
```

Comment:
The BDFL chapter above doesn't read like the BDFL needs to be involved in such decisions.
> Because rarely used, we refer to BDFL’s final authority as a “special” or “overriding”
vote.

Veto yes, but the "ratified by [...] the BDFL" reads like its a must.

Reply:
We can remove BDFL from here, I agree 100% with your point and only wanted that to mean "ask to ensure there is no veto"